src/Model/MethodGo.cs

```diff
@@ -592,7 +592,7 @@ public bool IsPageable
                 // if the nextLinkName field in the swagger has a null value ("nextLinkName": null)
                 // then don't treat this operation as pageable.
                 var pageableExtension = Extensions[AzureExtensions.PageableExtension] as JContainer;
-                return (string)pageableExtension["nextLinkName"] != null;
+                return !string.IsNullOrWhiteSpace((string)pageableExtension["nextLinkName"]);

```

Comment:
if the value is null in the swagger what does pageableExtension["nextLinkName"] return ? null or empty string or "null" ? if null, will casting to string work ?

Reply:
It used to return null, now it returns empty string (I think this happened when the front end switched to TS).